backend/pyproject.toml

```diff
@@ -30,8 +30,8 @@ dependencies = [
     "python-socketio==5.9.0", # For log_events
     "social-auth-app-django==5.3.0", # For OAuth
     "social-auth-core==4.4.2", # For OAuth
-    "unstract-sdk~=0.18.0",
-    "unstract-adapters~=0.8.0",
+    "unstract-sdk==0.19.0",
```

Comment:
NIT: Any reason for pinning this and not leaving it as `~=`

Reply:
Thank you for catching. Was trying out a few things and forgot to change it here. Done now.